src/renderEmail.js

```diff
@@ -1,6 +1,12 @@
 import ReactDOMServer from 'react-dom/server'
 
-export default function renderEmail(emailComponent) {
+export default function renderEmail(emailComponent, { renderMailchimpAttr } = {}) {
   const doctype = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">'
-  return doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+  let markup = doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+
+  if (renderMailchimpAttr) {
```

Comment:
Great idea. I think your suggestion to have it as it's own module would be ideal. It's such a small change that anyone else would be able to use outside of react-html-email and might find it useful.